config/default/common/config/metadata/stories/black_marble_night_lights/step005.md

```diff
@@ -0,0 +1 @@
+The Black Marble Blue/Yellow Composite (Day/Night Band) layer is a false color composite using the VIIRS at-sensor radiance and the brightness temperatures from the M15 band and complements the At Sensor Radiance layer. This false color scheme produces nighttime city lights in shades of yellow with infrared, nighttime cloud presence in shades of blue. During bright moonlight conditions, moonlight reflected from cloud tops and the land surface may also provide a yellow hue to those features. This image shows Tropical Storm Mindulle on 30 September 2021 nearing Japan. [Learn more about the Blue/Yellow composite](https://earthdata.nasa.gov/learn/articles/black-marble-blue-yellow-composite)
```

Comment:
--> "The Black Marble Blue/Yellow Composite (Day/Night Band) layer is a false color composite using VIIRS at-sensor radiance combined with brightness temperature from the M15 band which complements the At Sensor Radiance layer."